frontend/src/entities/dataset/dataset.css

```diff
@@ -30,4 +30,5 @@
 .drop-box {
     border: dashed 1px;
     padding: 15px;
+    border-radius: var(--border-radius-container-wqv1zi);
```

Comment:
I don't know much about styling cloudscape styling, how sure are we that this variable is stable?

Reply:
I spoke with the cloudscape team and it seems I should be using design tokens instead: https://cloudscape.design/foundation/visual-foundation/design-tokens/